test/system/520-checkpoint.bats

```diff
@@ -22,6 +22,13 @@ function setup() {
         skip "checkpoint does not work rootless"
     fi
 
+    # As of 2024-05, crun on Debian is not built with criu support
```

Comment:
might want to drop a link https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1008249
Not that I think this will be fixed anytime soon